testbed/tests/app/test_desktop.py

```diff
@@ -193,21 +195,33 @@ async def test_full_screen(app, app_probe):
 
         window1.show()
         window2.show()
-        await app_probe.redraw("Extra windows are visible")
-
-        assert not app.is_full_screen
+        # Add delay for gtk to show the windows
+        await app_probe.redraw("Extra windows are visible", delay=0.1)
+        with pytest.warns(
+            DeprecationWarning,
+            match=r"`App.is_full_screen` is deprecated. Use `App.is_presentation_mode` instead.",
+        ):
+            assert not app.is_full_screen
```

Comment:
The testbed doesn't need to exercise deprecated APIs. Only the dummy/frontend has deprecations. The testbed should only ever be touching current APIs.

Reply:
Done.